client/src/components/JobInformation/JobInformation.vue

```diff
@@ -88,79 +44,34 @@
 <script>
 import { mapCacheActions } from "vuex-cache";
 import { getAppRoot } from "onload/loadConfig";
-import { getGalaxyInstance } from "app";
-import { Services } from "./services";
+import DecodedId from "../DecodedId.vue";
+import CodeRow from "./CodeRow.vue";
 
 export default {
-    data() {
-        return {
-            decoded_history_id: undefined,
-            decoded_job_id: undefined,
-            decoded_copied_from_job_id: undefined,
-        };
+    components: {
+        CodeRow,
+        DecodedId,
     },
     props: {
-        hda_id: {
-            type: String,
-            required: true,
-        },
```

Comment:
Passed John convinced himself there were times the Job API would reject requests for information but that if the an accessible HDA was passed it made sense to render the information. There must be some comments about it server side somewhere? I hope you have thought through all of that. Like maybe this component is sitting on a published Page (it can be used from the Markdown engine) with a public HDA that doesn't belong to the user but that the viewer of the page should be able to see? Maybe this wouldn't work in that case anyway, no promises either way.

Reply:
The github UI didn't get updated, but this was merged with https://github.com/galaxyproject/galaxy/pull/10980. I'll see what happens. In case this doesn't work, do you think we can consider a job accessible if any one of its outputs is accessible ?

Reply:
Who merged that without my review.... ... oh 🤣 .

Umm.. I think effectively it was - if you want to formalize that by just allowing access by job_id I think that is fine. I was clearly being very conservative here - because it is a thorny question and I didn't want to make any decisions, just preserve what was working.